connmgr.go

```diff
@@ -313,15 +313,13 @@ func (cm *BasicConnMgr) getConnsToClose() []network.Conn {
 		return nil
 	}
 
-	nconns := int(atomic.LoadInt32(&cm.connCount))
-	if nconns <= cm.cfg.lowWater {
+	if int(atomic.LoadInt32(&cm.connCount)) <= cm.cfg.lowWater {
```

Comment:
Collapsed into one line to convince myself that `nconns` is not used after this, as this would be racy.